web/modules/custom/custom_page/config/install/core.entity_form_display.node.custom_page.default.yml

```diff
@@ -15,20 +16,28 @@ mode: default
 content:
   body:
     type: text_textarea_with_summary
-    weight: 1
+    weight: 2
     settings:
       rows: 9
       summary_rows: 3
       placeholder: ''
     third_party_settings: {  }
+  og_group_ref:
+    type: og_complex
+    weight: 0
+    settings:
+      match_operator: CONTAINS
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
```

Comment:
I would definitely hide the OG Complex field. Nobody should have access to this, we manage the group content through the "Add custom page" button instead.